packages/labs/context/src/legacy-decorators.ts

```diff
@@ -0,0 +1,2 @@
+export * from './lib/decorators/consume.js';
```

Comment:
add a comment about keeping this in sync with decorators.ts?

and do we need a license header?